pkgs/applications/misc/houdini/default.nix

```diff
@@ -1,22 +1,45 @@
-{ lib, stdenv, writeScript, callPackage, buildFHSEnv, unwrapped ? callPackage ./runtime.nix {} }:
-
-buildFHSEnv rec {
+{ lib, stdenv, writeScript, callPackage, buildFHSEnvChroot, bubblewrap, unwrapped ? callPackage ./runtime.nix {} }:
+let
+  make_empty_ocl_in_etc = stdenv.mkDerivation{  # a cheat to have OpenCL/vendors dir to bind at runtime
+    name = "make_empty_ocl_in_etc";
+    buildCommand = ''
+      mkdir -p $out/etc/OpenCL/vendors
+    '';
+  };
+in
+buildFHSEnvChroot rec {
   name = "houdini-${unwrapped.version}";
 
   targetPkgs = pkgs: with pkgs; [
     libGLU libGL alsa-lib fontconfig zlib libpng dbus nss nspr expat pciutils
-    libxkbcommon libudev0-shim tbb
+    libxkbcommon libudev0-shim tbb xwayland qt5.qtwayland nettools
+    bintools  # needed for ld and other tools, so ctypes can find/load sos from python
+    ocl-icd  # needed for opencl
+    numactl ncurses5  # needed by hfs ocl backend
   ] ++ (with xorg; [
-    libICE libSM libXmu libXi libXext libX11 libXrender libXcursor libXfixes
+    libICE libSM libXmu libXi libXt libXext libX11 libXrender libXcursor libXfixes
     libXrender libXcomposite libXdamage libXtst libxcb libXScrnSaver
-  ]);
+    libXrandr
+    libxcb xcbutil xcbutilimage xcbutilrenderutil xcbutilcursor xcbutilkeysyms xcbutilwm
+  ]) ++ [
+    make_empty_ocl_in_etc
+  ];
```

Comment:
These are becoming unwieldy, perhaps now would be a good time to split this across multiple lines.

Reply:
what do you suggest? having one per line seem excessive and less readable

Reply:
One per line is what people prefer these days.

IMO it creates _much_ more readable diffs and I don't think it hurts readability. If I see a long vertical stack of package names, I immediately recognise it as a list of buildInputs and can trivially mentally skip it if I don't care about buildInputs right now.

Reply:
Best try it for yourself. If you're going to do that, please do it in a separate commit so it's easy to tell which deps you added.